web/src/components/TestSpecForm/AssertionCheck.tsx

```diff
@@ -50,6 +64,18 @@ export const AssertionCheck = ({field, index, name, remove, form, assertions, at
     [assertions, attributeList, form, name]
   );
 
+  const selector = Form.useWatch('selector') || '';
+  const editorContext = useMemo(() => {
+    const [spanId = ''] = spanIdList;
```

Comment:
Why do we need the first `spanId`?

Reply:
Yeah, I'm going to add a TODO here, we have the same issue as when we auto-complete the expected value with first span info, even if there can be something from a different one. I'm going to bring this to the rest of the team.